src/bp/ui-studio/src/web/views/OneFlow/diagram/nodes/SaySomethingNode/index.tsx

```diff
@@ -22,6 +24,15 @@ export class SaySomethingWidget extends Component<{
     return (
       <div
         onClick={() => this.props.diagramEngine.flowBuilder.props.switchFlowNode(this.props.node.id)}
+        onContextMenu={e => {
+          contextMenu(
+            e,
+            <Menu>
+              <MenuItem text="Rename" onClick={() => console.log('delete')} />
```

Comment:
I guess those console.log will go away in subsequent PR ?

Reply:
Was just for testing, forgot to cleanup 🤦